Create a blog post about the sprint 172 (#472)
openSUSE/obs-landing: _posts/development/2024-08-27-content-moderation.md

---
layout: post

> title: "Improving moderators' ability to stay informed about the reports"

Reviewer: ```suggestion
title: "Improving notifications to stay informed about moderation reports"
```

Author: Applied

category: development
---

> In the latest set of improvements coming to OBS content moderation feature we focused on improving the experience of moderators reviewing incoming reports, to make it easier for them to comprehend the state of users and reported content in the notifications themselves, which will hopefully make it easier and faster for the moderators to respond to reports coming from users.

Reviewer: ```suggestion
In the latest set of improvements coming to OBS content moderation feature we focused on improving the experience of moderators reviewing incoming reports. By including more information about the state of reports we make it easier for moderators to comprehend the user reported content in the notifications. This will lower the time the moderators need to respond to reports.
```

> In the latest set of improvements coming to OBS content moderation feature we focused on improving the experience of moderators reviewing incoming reports. By including more information about the state of reports we make it easier for moderators to comprehend the user reported content in the notifications. This will lower the time the moderators need to respond to reports.

Reviewer: ```suggestion
In the latest set of improvements coming to the content moderation feature we focused on improving the experience of moderators reviewing incoming reports. By including more information about the state of reports we make it easier for moderators to comprehend the user reported content in the notifications. This will lower the time the moderators need to respond to reports.
```
